Phonics/View Controllers/LettersViewController.swift

```diff
@@ -18,6 +19,8 @@ class LettersViewController: UIViewController, UICollectionViewDataSource, UICol
     static func present(from source: UIViewController, with difficulty: Letter.Difficulty) {
         let controller = UIStoryboard(name: "Main", bundle: nil).instantiateViewController(withIdentifier: storyboardId) as! LettersViewController
         controller.difficulty = difficulty
+        controller.phonics = PHContent.allPhonicsSorted()
```

Comment:
`PHContent` is a singleton, so `PHContent.allPhonicsSorted` is always available in any scope. No need for a `controller.phonics` instance variable.